e2e/pages/courses.po.js

```diff
@@ -30,6 +32,22 @@ class Courses {
         });
         return this;
     }
+
+    deleteCourse(courseName){
+        CommonActions.click(`//span[@class='course-title' and text()='${courseName}']/ancestor::li/descendant::span[contains(@class, 'action-links-unfold-text')]`);
+        CommonActions.click(`//span[@class='course-title' and text()='${courseName}']/ancestor::li/descendant::a[text()='Delete']`);
+        CommonActions.click('input#edit-submit');
+        return this;
+    }
+
+    getDeleteConfirmationMessage(){
```

Comment:
may be better something like: getConfirmationMessage(), since the message when you delete, update a course will be displayed in same place